docs/dev-guide/tscript/debugging.mdx

```diff
@@ -190,13 +190,20 @@ The two main sources of dev-prod discrepancies are in bundling and connecting.
 
 #### Production bundling
 
+You may experience your Client sending stripped names as the Workflow "Type" when scheduling a Workflow.
 If you are getting errors like this:
 
 ```
 Error: 3 INVALID_ARGUMENT: WorkflowType is not set on request.
 ```
 
-This is due to your bundler stripping out Workflow function names, which we rely on to set the "Workflow Type" in Temporal. Turn it off and it should work.
+Or, if you are seeing stripped-down Workflow types in the Temporal Service's Web UI like this:
```

Comment:
I'm not sure there's enough context here - the alt text of the image actually does explain what we're looking for, but I'd take some of that prose and explain what's specifically different here. 

Webpack changed the Workflow's function name to something shorter, and now Temporal doesn't know how to handle it.

Reply:
Updated as suggested